### PR TITLE
removed lt and gt as operators

### PR DESCRIPTION
**Pull request by mburke5678**
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> For 1.7 Features: set Milestone to `1.7` and Base Branch to `release-1.7`
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>
> NOTE: Please check the “Allow edits from maintainers” box below to allow 
> reviewers fix problems on your patch and speed up the review process.
> Please delete this note before submitting the pull request.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/4152)

### CODE REVIEW

**Comment by mburke5678** [comment]
@aveshagarwal PTAL

File: docs/concepts/configuration/assign-pod-node.md [hunk 1]
```diff
@@ -132,7 +132,7 @@ This node affinity rule says the pod can only be placed on a node with a label w
 among nodes that meet that criteria, nodes with a label whose key is `another-node-label-key` and whose
 value is `another-node-label-value` should be preferred.
 
-You can see the operator `In` being used in the example. The new node affinity syntax supports the following operators: `In`, `NotIn`, `Exists`, `DoesNotExist`, `Gt`, `Lt`.
+You can see the operator `In` being used in the example. The new node affinity syntax supports the following operators: `In`, `NotIn`, `Exists`, `DoesNotExist`.
```
Comment (aveshagarwal) [inline docs/concepts/configuration/assign-pod-node.md]: Node affinity works with all 6 operators: `In`, `NotIn`, `Exists`, `DoesNotExist`, `Gt`, `Lt` . pod affinity/anti-affinity only works with regular 4 operators: `In`, `NotIn`, `Exists`, `DoesNotExist`.

**Comment by mburke5678** [comment]
@aveshagarwal Apologies. I replaced `lt' and `gt` for node-affinity.

Original change per [4137](https://github.com/kubernetes/kubernetes.github.io/issues/4137)

File: docs/concepts/configuration/assign-pod-node.md [hunk 2]
```diff
@@ -188,7 +188,7 @@ label having key "security" and value "S2".) See the [design doc](https://github
 for many more examples of pod affinity and anti-affinity, both the `requiredDuringSchedulingIgnoredDuringExecution`
 flavor and the `preferredDuringSchedulingIgnoredDuringExecution` flavor.
 
-As with node affinity, the legal operators for pod affinity and anti-affinity are `In`, `NotIn`, `Exists`, `DoesNotExist`, `Gt`, `Lt`.
+As with node affinity, the legal operators for pod affinity and anti-affinity are `In`, `NotIn`, `Exists`, `DoesNotExist`.
```
Comment (aveshagarwal) [inline docs/concepts/configuration/assign-pod-node.md]: Not sure `As with node affinity` fits now as node affinities and pod affinities operators are not same.

Comment (bsalamat) [inline docs/concepts/configuration/assign-pod-node.md]: As @aveshagarwal mentioned, node affinity supports "Gt" and "Lt" as well. Pod affinity term uses standard label selectors which support only In, NotIn, Exists, and DoesNotExist. So, please remove `As with node affinity, `.

**Comment by mburke5678** [comment]
@bsalamat Good catch. Thank you. Made the change.

**Comment by bsalamat** [comment]
/lgtm

Thanks, @mburke5678 !

**Comment by mburke5678** [comment]
@bsalamat @aveshagarwal Can one of you merge? I do not have permissions, it seems.

**Comment by bsalamat** [comment]
I don't have write access either.
@davidopp can you approve?

**Pull request merged**